src/controllers/notifications.js

```diff
@@ -71,6 +73,16 @@ const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
 exports.post = function * postNotification () {
   const notification = yield requestUtil.validateBody(this, 'Notification')
   log.debug('Got notification: ' + JSON.stringify(notification))
+
+  const verifySignature = this.config.get('notifications.must_verify')
+  if (verifySignature) {
+    const result = model.verifySignature(notification, this.config)
+    if (!result.valid) {
+      log.warn('Signature verification failed: ' + result.error)
+      throw new UnprocessableEntityError('Notification failed signature verification')
```

Comment:
From, https://github.com/interledger/five-bells-connector/pull/125#discussion_r57749447

>  but let's remember we'll have a message flooding problem if this error condition gets triggered 

cc @naoitoi 
So, in reality, we should never have a message flooding problem.

Retrying infinite times is probably the right thing to do then. If we assume that owners of Ledgers, and other components are rational, knowledgeable operators. If a Connector is flooded with messages by a Ledger they trust, these are problems that the software can't fix.